templates/docs/content-guidelines.md

```diff
@@ -0,0 +1,209 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Content Guidelines
+  description: Content guidelines for the Canonical design system
+---
+
+Open source software connects and uplifts, and our content does the same. With a diverse and global community of users, our language should be clear, consistent, and accessible.
+
+Follow this overall guidance for all UX copy:
+
+- Put the most important information first
+- Use short words and short sentences
+- Use sentence case
+- Avoid "please" and "thank you" – in UI copy, they can vary by culture
+- Use inclusive language – avoid gendered pronouns and ableist words such as "see" or "view"
+
+## Abbreviations
+
+Do not use e.g. or i.e. Instead, use ‘for example’ or ‘meaning’.
+
+## Alt text for images
+
+Alt text is a short description that replaces an image for users who don’t see the image. They may rely on screen readers or other assistive technologies. This is also good for SEO.
+
+### How screen readers work
+
+Screen readers follow the HTML reading order to read and interpret content for users.
+
+When an image conveys important information, alt text is how users who don’t see the images consume that information. When a screen reader gets to an image, it will read the alt text to describe the content or purpose. Screen readers ignore images with empty alt attributes.
+
+For complex images like graphs or diagrams, consider putting a longer description somewhere else on the page. For example, in a nearby paragraph or through ARIA attributes.
+
+### When to write alt text
+
+- Always write alt text when the image adds meaning.
+- Use an empty alt attribute (`alt=""`) if the image doesn’t add meaning. For example, the currency icons below are decorative and do not add meaning. [Learn more about decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).
```

Comment:
> the currency icons below are decorative and do not add meaning

"currency icons" are mentioned but not included in the page. I think we are missing something from the copydoc here.

Reply:
@advl has added the image. resolving